Source/Loader/App.config

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
Whats the reason for adding this file? This doesn't seem neccessary.

Reply:
Sorry: after the checks this file turns out to be unnecessary. I will try to make a pull request without this file or you may try to accept it without this odd file.